nickparen.py

```diff
@@ -0,0 +1,17 @@
+def check_parentheses(string_to_test):
+    if not (isinstance(string_to_test, unicode) ^
+            isinstance(string_to_test, str)):
+        raise TypeError("This isn't a string!")
```

Comment:
a solid py2 way of testing this would be

```python
if not is instance(string_to_test, basestring):
    raise TypeError
```
That's not available in Py3 any longer, but the [six](https://pythonhosted.org/six/) module gives you the `six.string_types` which would pass in Python 2 for both `str` and `unicode` objects:

```python
import six
if not is instance(string_to_test, six.string_types):
    raise TypeError